docker/README.md

```diff
@@ -288,6 +286,10 @@ Notes on a couple of the parameters:
 
     --log-level
     optional
+    
+    ## present only in [mms_app_gpu.conf](./mms_app_gpu.conf)
```

Comment:
This markdown syntax wouldn't work in this section. Probaly change it to" Following option is used only for GPU"